examples/gallery/lines/roads.py

```diff
@@ -14,8 +14,10 @@
 # %%
 import geopandas as gpd
 import pygmt
+import pyogrio
```

Comment:
Can we move it to `conf.py` so that we don't have to repeat it in multiple gallery examples?

Reply:
Just tried locally, no it doesn't work if I put it in `doc/conf.py`.

Reply:
I'm hesitating to merge this PR because (1) users may be confused about `pyogrio.core._register_drivers()`; (2) Users may still use geopandas v0.x and don't have pyogrio installed.

Reply:
In commit e15462f956c5ee11d0cf01efe14c4e79edb0f4bf, I use sphinx-gallery's [`sphinx_gallery_start_ignore`/`sphinx_gallery_end_ignore`](https://sphinx-gallery.github.io/stable/configuration.html#hiding-code-blocks) flags to hide the two lines of codes in the documentation.

**Preview**: https://pygmt-dev--3305.org.readthedocs.build/en/3305/gallery/lines/roads.html#sphx-glr-gallery-lines-roads-py
https://pygmt-dev--3305.org.readthedocs.build/en/3305/get_started/04_table_inputs.html#sphx-glr-get-started-04-table-inputs-py